etc/ReplayOfflineConfiguration.py

```diff
@@ -113,7 +113,7 @@
 setScramArch(tier0Config, "CMSSW_12_3_0", "cs8_amd64_gcc10")
 
 # Configure scenarios
-ppScenario = "ppEra_Run3"
+ppScenario = "ppEra_Run3_2023"
```

Comment:
there are a few other scenarios here below that use `"ppEra_Run3"`:
- ppScenarioB0T = "ppEra_Run3"
- hiScenario = "ppEra_Run3"
- hiTestppScenario = "ppEra_Run3"

Should we review/update those as well?

Reply:
As far as I know, not for the Heavy Ion ones. Maybe we could start with testing only the ppScenario which is the most urgent in any case.